curriculum/challenges/english/20-upcoming-python/learn-tree-traversal-by-building-a-binary-search-tree/655a54cd88e33b8646c67e16.md

```diff
@@ -0,0 +1,89 @@
+---
+id: 655a54cd88e33b8646c67e16
+title: Step 46
+challengeType: 20
+dashedName: step-46
+---
+
+# --description--
+
+Within the `inorder_traversal` method, initiate the in-order traversal by calling the helper method `_inorder_traversal` and pass the BST root and the `result` list as the arguments.
+
+It will start the traversal from the root of the binary search tree (`self.root`), and the result list will be passed to accumulate the keys during the traversal.
```

Comment:
```suggestion
This will start the traversal from the root of the binary search tree (`self.root`), and the `result` list will be passed to accumulate the keys during the traversal.
```